nssp_demo/forecast_non_target_visits.R

```diff
@@ -99,7 +99,7 @@ n_forecast_days <- argv$n_forecast_days
 n_samples <- argv$n_samples
 
 disease_name_nssp_map <- c(
-  "covid-19" = "COVID-19/Omicron",
+  "covid-19" = "COVID-19",
```

Comment:
I thought we had agreed that our new vintages would use `COVID-19/Omicron` to be consistent with the modern vintages?

Reply:
Vintages yes, but in the version of `prep_data.py` in this PR I change `COVID-19/Omicron` to `COVID-19` so we can use `COVID-19` throughout downstream.